perf/pulsar-perf-go.go

```diff
@@ -0,0 +1,49 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+	log "github.com/sirupsen/logrus"
```

Comment:
we should add package management tool. `go mod`, `dep`, `glide` and so on. I recommend using `go mod`

Reply:
Surely, I left it over for now since it's really a WIP and I'm not superfamiliar with these tools. Perfectly fine to go with `go mod`.

Reply:
ok, when this pull request is merged, I will add it.

Reply:
Another question is, do we need to add the `vendor` directory, after executing `go mod`, all the dependencies will be stored in the `vendor` directory. The advantage of adding the vendor directory is that when the code repository we reference is deleted, we can also maintain our own local version. Of course, this is not a standard. Some projects have uploaded vendor. For example: [etcd](https://github.com/etcd-io/etcd) Some items are not uploaded, for example: [TiDB](https://github.com/pingcap/tidb). so, in our project, do we upload the vendor?

Reply:
I'd prefer to only check-in code that is manually written for this library in this repo. Of course there are always exception (eg: the protobuf generated code, for ease of use), but if the downloading of specific tag of a dependency can be automated, that seems a better solution.

We could rethink that if we see that tags are being removed upstream.

Reply:
+1